spdlog/src/logger.rs

```diff
@@ -470,6 +470,14 @@ impl Logger {
         })
     }
 
+    pub(crate) fn flush_sinks_atexit(&self) {
+        self.sinks.iter().for_each(|sink| {
+            if let Err(err) = sink.flush_atexit() {
+                self.handle_error(err);
+            }
+        });
+    }
+
```

Comment:
This may be even `pub` if users set their own global static LOGGER. But given that is not quite possible it's OK to leave `pub(crate)` until real use cases occur.